test/plot.js

```diff
@@ -38,6 +38,10 @@ import * as plots from "./plots/index.js";
         }
 
         test.ok(actual === expected, `${name} must match snapshot`);
+        if (actual !== expected) {
+          const outfile = path.resolve("./test/output", path.basename(name, ".js") + "-changed." + ext);
```

Comment:
It might be nice to console.log here the outfile so that it’s obvious to folks where they can find the generated output.